sample-apps/django_sample/django_sample/settings.py

```diff
@@ -47,7 +47,7 @@
     # sent to a mock endpoint.
     "API_BASE_URL": "https://judoscale-python.requestcatcher.com",
     # "LOG_LEVEL": "DEBUG",
-    "REPORT_INTERVAL_SECONDS": 15,
+    # "REPORT_INTERVAL_SECONDS": 5,
```

Comment:
Just using the default interval for simplicity, and 10 seconds is better when testing changes than 15.